configs/sites/tier1/orion/packages_oneapi.yaml

```diff
@@ -0,0 +1,52 @@
+packages:
+  all:
+    compiler:: [oneapi@2024.2.1]
+    providers:
+      mpi:: [intel-oneapi-mpi@2021.13]
+      # Remove the next three lines to switch to intel-oneapi-mkl
+      blas:: [openblas]
+      fftw-api:: [fftw]
+      lapack:: [openblas]
+  mpi:
+    buildable: False
+  intel-oneapi-mpi:
+    buildable: False
+    externals:
+    - spec: intel-oneapi-mpi@2021.13%oneapi@2024.2.1
+      modules:
+      - spack-managed-x86-64_v3
+      - intel-oneapi-compilers/2024.2.1
+      - intel-oneapi-mpi@2021.13.1
+      prefix: /apps/spack-managed-x86_64_v3-v1.0/oneapi-2024.2.1/intel-oneapi-mpi-2021.13.1-3pv63eugwmse2xpeglxib4dr2oeb42g2
+
+  intel-oneapi-mkl:
+    buildable: False
+    externals:
```

Comment:
Does it make sense to comment out lines 24-29 if you don't use MKL? Can you check if MKL shows up somewhere in the concretizer log?

Reply:
here:
```
 -   crmijvv          ^gsibec@1.2.1%oneapi@2024.2.1~ipo~mkl build_system=cmake build_type=Release generator=make arch=linux-rocky9-skylake_avx512
 -   c3wec4p      ^ectrans@1.5.0%oneapi@2024.2.1+double_precision+fftw~ipo~mkl+mpi+openmp+single_precision+transi build_system=cmake build_type=RelWithDebInfo generator=make patches=1799948 arch=linux-rocky9-skylake_avx512
```